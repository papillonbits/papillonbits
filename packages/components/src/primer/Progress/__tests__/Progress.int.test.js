import { normal, large, small } from './Progress.int.story'

describe('<Progress />', () => {
  describe('Render', () => {
    test('must match normal()', () => {
      expect(global.renderToJSON(normal())).toMatchSnapshot()
    })

    test('must match large()', () => {
      expect(global.renderToJSON(large())).toMatchSnapshot()
    })

    test('must match small()', () => {
      expect(global.renderToJSON(small())).toMatchSnapshot()
    })
  })
})
