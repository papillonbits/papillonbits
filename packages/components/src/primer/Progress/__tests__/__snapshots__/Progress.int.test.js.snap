// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Progress /> Render must match large() 1`] = `
<span
  className="Progress Progress--large"
>
  <span
    className="Progress-item bg-green"
    style={
      Object {
        "width": "50%",
      }
    }
  />
</span>
`;

exports[`<Progress /> Render must match normal() 1`] = `
<span
  className="Progress"
>
  <span
    className="Progress-item bg-green"
    style={
      Object {
        "width": "50%",
      }
    }
  />
</span>
`;

exports[`<Progress /> Render must match small() 1`] = `
<span
  className="Progress Progress--small"
>
  <span
    className="Progress-item bg-green"
    style={
      Object {
        "width": "50%",
      }
    }
  />
</span>
`;
